README.md

```diff
@@ -1,11 +1,33 @@
-# Packetgraph
+# Packetgraph ?
 
-Packetgraph library is build upon [DPDK](http://dpdk.org/) making a collection
-of network bricks you can connect to form a network graph.
-The goal of this project is to provide a low-latency software defined
-network.
-Everyone is free to use this library to build up there own network
-infrastructure.
+Packetgraph library is build upon [DPDK](http://dpdk.org/) making a
+collection of network bricks you connect to form a graph.
+
+[![Build Status](https://travis-ci.org/outscale/packetgraph.svg?branch=master)](https://travis-ci.org/outscale/packetgraph)
+
+The goal of this library is to provide a really EASY interface to
+build you own DPDK based application using [Network Function
+Virtualization](https://en.wikipedia.org/wiki/Network_function_virtualization)
+Everyone is free to use this library to build up there own network application.
+
+Once you have created and connected all bricks in you network graph,
+some bricks will be able to poll a burst of packets (max 64 packets)
+and let the burst propagate in you graph.
+
+Connections between bricks don't store any packets and each burst will
+propagate in the graph without any copy.
+
+Each graph run on one core but you can connect different graph using
+Queue bricks (which are thread safe). For example, a graph can be
+split on demand to be run on different core or even merged.
+
+To show how look like a graph, it is possible to generate a graph
```

Comment:
"To show how look like a graph" fell a little odd.
"If you want a graphical representation of a graph..."